source/reference/oplog-sizing.txt

```diff
@@ -16,11 +16,12 @@ The :ref:`mongosync <c2c-mongosync>` program uses :ref:`change streams
 <changeStreams>` to synchronize data between source and destination
 clusters. ``mongosync`` does not access the :term:`oplog` directly,
 but when a change stream returns events from the past, the events must
-be within the :term:`oplog` time range. 
+be within the ``oplog`` time range. 
 
-.. include:: /includes/fact-oplog-background.txt
 
-Estimate ``oplog`` Size Needed for Initial Sync
+.. include:: /includes/fact-oplog-background
+
+Monitor ``oplog`` Size Needed for Initial Sync
```

Comment:
([titles](https://www.mongodb.com/docs/meta/style-guide/style/titles-and-headings/))
It's fine to omit font treatment from titles and headings.

Reply:
Fixed